### PR TITLE
the img-paste script don't work with wsl2 for me.

### PR DESCRIPTION
**Pull request by vertexi**
I just rewrite the wsl detection part and powershell script.

Now it's works to me.

### CODE REVIEW

File: plugin/mdip.vim [hunk 1]
```diff
@@ -29,12 +29,14 @@ endfunction
...
+    else
+      let tmpfile = '\\\\wsl\$\\Ubuntu'.tmpfile
+    endif
 
-    let clip_command = "Add-Type -AssemblyName System.Windows.Forms;"
-    let clip_command .= "if ([Windows.Forms.Clipboard]::ContainsImage()) {"
-    let clip_command .= "[Windows.Forms.Clipboard]::GetImage().Save(\\\""
-    let clip_command .= tmpfile ."\\\", [System.Drawing.Imaging.ImageFormat]::Png) }"
-    let clip_command = "powershell.exe -sta \"".clip_command. "\""
+    let clip_command = 'powershell.exe -sta "Add-Type -Assembly PresentationCore;\$img = [Windows.Clipboard]::GetImage();\$file = \"'. tmpfile . '\";\$stream = [IO.File]::Open(\$file, \"OpenOrCreate\");\$encoder = New-Object Windows.Media.Imaging.JpegBitmapEncoder;\$encoder.QualityLevel = 90;\$encoder.Frames.Add([Windows.Media.Imaging.BitmapFrame]::Create(\$img));\$encoder.Save(\$stream);\$stream.Dispose()"'
```
Comment (ferrine) [inline plugin/mdip.vim]: can you please make it more readable? One-liner is very hard

Comment (vertexi) [reply plugin/mdip.vim]: Done

**Comment by ferrine** [comment]
Much better now. It's it finished?

**Comment by vertexi** [comment]
Now I add failure detection.

I think it works for me on the wsl2. So it's maybe finished

**Comment by ferrine** [comment]
Thanks

**Pull request merged**